firmware/common/cpld_xc2c.c

```diff
@@ -401,13 +477,22 @@ bool cpld_xc2c64a_jtag_sram_verify(
 	 * the first row's data.
 	 */
 	bool matched = true;
-	for(size_t address_row=0; address_row<=CPLD_XC2C64A_ROWS; address_row++) {
-		const int data_row = (int)address_row - 1;
-		const size_t mask_index = (data_row >= 0) ? verify->mask_index[data_row] : 0;
-		const uint8_t* const expected = (data_row >= 0) ? &program->row[data_row].data[0] : NULL;
-		const uint8_t* const mask = (data_row >= 0) ? &verify->mask[mask_index].value[0] : NULL;
-		const uint8_t next_address = (address_row < CPLD_XC2C64A_ROWS) ? cpld_hackrf_row_addresses.address[address_row] : 0;
-		matched &= cpld_xc2c64a_jtag_sram_compare_row(jtag, expected, mask, next_address);
+	for (size_t address_row = 0; address_row <= CPLD_XC2C64A_ROWS; address_row++) {
+		const int data_row = (int) address_row - 1;
+		const size_t mask_index =
+			(data_row >= 0) ? verify->mask_index[data_row] : 0;
+		const uint8_t* const expected =
+			(data_row >= 0) ? &program->row[data_row].data[0] : NULL;
+		const uint8_t* const mask =
+			(data_row >= 0) ? &verify->mask[mask_index].value[0] : NULL;
+		const uint8_t next_address = (address_row < CPLD_XC2C64A_ROWS) ?
+			cpld_hackrf_row_addresses.address[address_row] :
+			0;
```

Comment:
Oh, I guess it's because it split the ternary expression, and when it does so it always puts both results on their own lines. Which makes sense as a rule, but just looks weird here because the `0` is so much shorter than the other result.